src/main/java/Duke.java

```diff
@@ -1,10 +1,162 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    /**
+     * Number of tasks scheduled
+     */
+    public static int taskCounter = 0;
+
+    /**
+     * Task type 100 objects to store the tasks the user will create
+     */
+    public static Task[] scheduledTasks = new Task[100];
```

Comment:
Could replace the magic number (100) with a named constant.

Reply:
Sure, I will do that.